Assets/Scripts/IUnityService.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+public interface IUnityService
+{
+    float GetDeltaTime();
+    float GetAxisRaw(string axisName);
+    bool GetButton(string buttonName);
+}
+
+class UnityService : IUnityService
```

Comment:
Following the things I suggested above:
```suggestion
class UnityInputService : IInputService
```